cdap-docs/build.sh

```diff
@@ -395,20 +246,20 @@ function build_docs_inner_level() {
     echo "--------------------------------------------------------"
     echo
     cd $SCRIPT_PATH/${i}
-    ./build.sh ${1} ${ARG_2}
+    ./build.sh ${1}
```

Comment:
Since all of the `build.sh` arguments appear to map 1:1 to function calls, why don't use just call the functions, rather than having the script call itself with different arguments? That's really... dodgy.